schema/proto3/dhcp.proto

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+
+syntax = "proto3";
+
+package alcor.schema;
+
+option java_package = "com.futurewei.alcor.schema";
+option java_outer_classname = "DHCP";
+
+import "common.proto";
+
+message DHCPConfiguration {
+    int32 version = 1;
```

Comment:
Version shall be needed for other protobuf message as well.

Reply:
We have that for PortConfiguration, SubnetConfiguration, VpcConfiguration and now DHCPConfiguration. Do you mean to have version for *every* message including HostInfo or FixedIp under PortConfiguration? I think that would be a good idea.

Reply:
Versioning to the port level should be sufficient usually. It depends on which new scenarios could be covered for HostInfo and Fixed.